packages/rollup/src/executors/rollup/schema.d.ts

```diff
@@ -32,4 +32,6 @@ export interface RollupExecutorOptions {
   javascriptEnabled?: boolean;
   generateExportsField?: boolean;
   skipTypeCheck?: boolean;
+  babelConfig?: string;
+  babelUpwardRootMode?: boolean;
```

Comment:
Also need to add `babelUpwardRootMode` to the `schema.json` file. And I don't think we need `babelConfig` option here since Babel can find it automatically, we just want to let users enable or disable the root mode.